core/graph/src/test/java/uk/gov/gchq/gaffer/operation/export/graph/ExportToOtherAuthorisedGraphTest.java

```diff
@@ -55,6 +55,17 @@ public void builderShouldCreatePopulatedOperation() {
     }
 
     @Override
+    public void shouldShallowCloneOperation() {
+        // When
+        ExportToOtherAuthorisedGraph clone = (ExportToOtherAuthorisedGraph) op.shallowClone();
+
+        // Then
+        assertEquals("graphId", clone.getGraphId());
```

Comment:
For all these tests please can you also add:

assertNotSame(op, clone);

This will check that the instance is actually different.